cognite/replicator/replication.py

```diff
@@ -155,6 +162,7 @@ def make_objects_batch(
     project_src: str,
     replicated_runtime: int,
     depth: Optional[int] = None,
+    dst_ts: Optional[TimeSeries] = None,
```

Comment:
The parameter should be documented in the docstring.

Reply:
Done